packages/phrases/src/locales/de/translation/admin-console/sign-in-exp.ts

```diff
@@ -92,9 +92,11 @@ const sign_in_exp = {
   },
   custom_css: {
     title: 'CUSTOM CSS', // UNTRANSLATED
-    css_code_editor_title: 'Custom CSS to change UI', // UNTRANSLATED
-    css_code_editor_description: 'Description - Doc. <a>{{link}}</a>', // UNTRANSLATED
-    css_code_editor_description_link_content: 'Readme', // UNTRANSLATED
+    css_code_editor_title: 'Personalize your UI with Custom CSS', // UNTRANSLATED
+    css_code_editor_description: 'See the example of Custom CSS. <a>{{link}}</a>', // UNTRANSLATED
+    css_code_editor_description_link_content: 'Learn more', // UNTRANSLATED
+    css_code_editor_content_placeholder:
+      '// Enter your Custom CSS code to tailor the styles of color, font, components, and layout... of Sign-in, Create-account, Forgot-Password, and other pages to your exact specifications. Express your creativity and make your UI stand out.', // UNTRANSLATED
```

Comment:
It would be better to provide some code examples, like the one we mentioned in the rc.3 release notes.
```
Since Logto uses CSS Modules, you may see a hash value in the class property of DOM elements (e.g. a <div> with vUugRG_container). To override these, you can use the $= CSS selector to match elements that end with a specified value. In this case, it should be div[class$=container].
```

Reply:
We attached this complete example in the link.